packages/search-input/src/SearchInput.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import SearchInput from '.';
+import { axe } from 'jest-axe';
+import { SizeVariant } from './types';
+
+function renderSearchInput(props = {}) {
+  const utils = render(
+    <SearchInput
+      data-testid="search-input"
+      aria-labelledby="test-search-input"
+      {...props}
+    />,
+  );
+  const searchInput = utils.getByTestId('search-input');
+  return { ...utils, searchInput };
+}
+
+describe('packages/search-input', () => {
```

Comment:
Should probably replicate all the `text-input` tests

Reply:
Some tests aren't relevant for SearchInput (i.e. label, description, baseFontSize, etc.) so there won't be full parity, but I went back and added some more!